File: auto-docs/executables/python/axes_lines.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')

trace1 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[8, 7, 6, 5, 4, 3, 2, 1, 0]
)
trace2 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[0, 1, 2, 3, 4, 5, 6, 7, 8]
)
data = Data([trace1, trace2])
layout = Layout(
    xaxis=XAxis(
        showline=True,
        linecolor='#636363',
        linewidth=6,
        showgrid=True,
        gridcolor='#bdbdbd',
        gridwidth=2,
        zeroline=True,
        zerolinecolor='#969696',
        zerolinewidth=4,
        mirror='ticks'
    ),
    yaxis=YAxis(
        showline=True,
        linecolor='#636363',
        linewidth=6,
        showgrid=True,
        gridcolor='#bdbdbd',
        gridwidth=2,
        zeroline=True,
        zerolinecolor='#969696',
        zerolinewidth=4,
        mirror='ticks'
    )
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='axes-lines', auto_open=False)

File: auto-docs/executables/nodejs/axes_lines.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var trace1 = {
  x: [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  y: [8, 7, 6, 5, 4, 3, 2, 1, 0], 
  type: "scatter"
};
var trace2 = {
  x: [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  y: [0, 1, 2, 3, 4, 5, 6, 7, 8], 
  type: "scatter"
};
var data = [trace1, trace2];
var layout = {
  xaxis: {
    showgrid: true, 
    zeroline: true, 
    showline: true, 
    gridcolor: "#bdbdbd", 
    gridwidth: 2, 
    zerolinecolor: "#969696", 
    zerolinewidth: 4, 
    linecolor: "#636363", 
    linewidth: 6, 
    mirror: "ticks"
  }, 
  yaxis: {
    showgrid: true, 
    zeroline: true, 
    showline: true, 
    gridcolor: "#bdbdbd", 
    gridwidth: 2, 
    zerolinecolor: "#969696", 
    zerolinewidth: 4, 
    linecolor: "#636363", 
    linewidth: 6, 
    mirror: "ticks"
  }
};

var graph_options = {filename: "axes-lines", fileopt: "overwrite", layout: layout, auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});